update entry point articles in ag for android (#646)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/adguard-for-android.md

## Features

Main features and settings available in AdGuard for Android

[Features](/adguard-for-android/features/features.md)

## Installation

> Instructions on how to install and uninstall AdGuard for Android, as well as system requirements

Reviewer: может вот так?
```suggestion
Installation, removal, and system requirements for AdGuard for Android
```
